desktop/ui/src/main/java/org/datacleaner/widgets/tree/SchemaTree.java

```diff
@@ -108,13 +108,14 @@
     private final WindowContext _windowContext;
     private final AnalysisJobBuilder _analysisJobBuilder;
     private final InjectorBuilder _injectorBuilder;
+    private boolean _includeLibraryNode = true;
 
     private DatastoreConnection _datastoreConnection;
     private String _searchTerm = "";
 
     @Inject
     protected SchemaTree(final Datastore datastore, @Nullable AnalysisJobBuilder analysisJobBuilder,
-            WindowContext windowContext, InjectorBuilder injectorBuilder) {
+            WindowContext windowContext, InjectorBuilder injectorBuilder) throws Exception {
```

Comment:
We have a datastore example with salesforce that doesn't have the password set. Normally when we try to use this datastore in the main window it will throw an exception about it's missing password. My eclipse was too slow and I did not throw the exception in the ui. I ran the jar and it seems to work fine without the constructor throwing the exception. Therefore I removed the unnecessary code